moto/ec2/models.py

```diff
@@ -1047,6 +1049,7 @@ def attach_eni(self, eni, device_index):
         # This is used upon associate/disassociate public IP.
         eni.instance = self
         eni.attachment_id = random_eni_attach_id()
+        eni.attach_time = utc_date_and_time()
         eni.device_index = device_index
 
```

Comment:
it will be nice to update `eni.status = "in-use"` here as well, to reflect the ENI is in use by an instance once it's attached.

Reply:
Good idea, added